include/event2/thread.h

```diff
@@ -212,6 +212,26 @@ int evthread_use_windows_threads(void);
     @return 0 on success, -1 on failure. */
 EVENT2_EXPORT_SYMBOL
 int evthread_use_pthreads(void);
+
+/**
+   A flag used to describe which features Pthreads locking feature must provide.
+*/
+enum evthread_pthreads_flag {
```

Comment:
just `evthread_flags`

Reply:
also when you are using `enum` for flags you need to cast them (for bitwise and so on), let's make it just `#define ...` ?